chipsec/logger.py

```diff
@@ -262,6 +265,16 @@ def set_autolog_file(self):
         else:
             print('Unable to autolog')
 
+    def log_csv(self, file_name, test_cases):
+        fields = ['name', 'result', 'code', 'output']
+        if '.csv' not in file_name:
```

Comment:
should probably be `if not file_name.endswith('.csv'):` That will make sure the .csv is at the end of the string.